InformationScripting/src/query_framework/QueryResultVisualizer.cpp

```diff
@@ -113,11 +119,24 @@ Optional<int> QueryResultVisualizer::visualize(const TupleSet& ts)
 			item->addOverlay(overlay, HIGHLIGHT_OVERLAY_GROUP);
 		}
 	}
+
+	auto htmlTuples = ts.tuples("html");
+	if (htmlTuples.size()) {
+		QString url = (*htmlTuples.begin())["html"];
+		// TODO an overlay or something would probably be better for this:
+		// TODO position it a bit better?
+		Comments::VCommentBrowser* browser = new Comments::VCommentBrowser{nullptr,
```

Comment:
Theoretically, yes. But this is already broken in a different place: after cyril's work we now have a dependency on comments from within visualization base, but comments also depends on visualization base so that's a cycle. This works on linux, but is conceptually broken and won't work on Windows. So I might at some point just move the comment browser directly inside visualization base.

So I guess, for now, just leave the code as it is and we'll fix it later (hopefully soon).